src/main/kotlin/com/expedia/graphql/schema/hooks/SchemaGeneratorHooks.kt

```diff
@@ -14,6 +14,7 @@ import kotlin.reflect.KType
  * Hooks are lifecycle events that are called and triggered while the schema is building
  * that allow users to customize the schema.
  */
+@Suppress("Detekt.ComplexInterface")
```

Comment:
I think `ComplexInterface` and `TooManyFunctions` should also be disabled, the issues are not something you'd need a linter for and are often very much on purpose. Detekt and klint have such aggressive settings that I think it adds `@Supress` noise to the code.

Reply:
Yea, I agree. Since I deleted the detekt file here anyway I will add those rules in this PR